engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/filters/available-errata-filter.controller.js

```diff
@@ -83,6 +83,12 @@ angular.module('Bastion.content-views').controller('AvailableErrataFilterControl
             nutupane.refresh();
         };
 
+        $scope.$watch('rule.date_type', function (value) {
```

Comment:
If the function is defined in the html, and no function exists on the controller, no errors should be thrown. So you should be safe to define ng-change on the shared template.